third_party/blink/renderer/modules/ml/webnn/mojo_model_info.cc

```diff
@@ -1026,12 +1026,14 @@ size_t MojoModelInfo::AddOperandToModel(const MLOperand* output) {
   auto desc = ml::webnn::mojom::blink::OperandDescriptor::New();
   desc->data_type = BlinkOperandTypeToMojo(output->Type());
   desc->dimensions = output->Dimensions();
-  // Add operand descriptor into model.
-  model_info_->operands.push_back(std::move(desc));
-  // The index used to identify operand on the server side, each operation
-  // generate a output operand that will be inserted in a hash map with the
-  // MLOperand and index, the index is incremented by one.
-  size_t output_index = model_info_->operands.size() - 1;
+  // Add operand descriptor into model. The index used to identify operand on
+  // the server side, each operation generate a output operand that will be
+  // inserted in a hash map with the MLOperand and index, the index is
+  // incremented by one.
+  // Notice that, due to the limitation of WTF::HashMap (not allow 0 as key),
+  // the index is counted from 1;
+  size_t output_index = model_info_->operands.size();
+  model_info_->operands.insert(++output_index, std::move(desc));
   operand_index_map_.insert(output, output_index);
```

Comment:
```suggestion
  size_t output_index = model_info_->operands.size() + 1;
  model_info_->operands.insert(output_index, std::move(desc));
  operand_index_map_.insert(output, output_index);
```

(minor) It would be slightly more robust to set output_index + 1 to begin with rather than inside another stament via ++, just in case any of these lines get extended/reordered in the future.

Reply:
Update, thanks! @fdwr